var/spack/repos/builtin/packages/netcdf-fortran/package.py

```diff
@@ -35,3 +35,16 @@ class NetcdfFortran(AutotoolsPackage):
     version('4.4.3', 'bfd4ae23a34635b273d3eb0d91cbde9e')
 
     depends_on('netcdf')
+
+    @property
+    def libs(self):
+        libraries = ['libnetcdff']
+
+        # This package installs both shared and static libraries. Permit
+        # clients to query which one they want.
+        query_parameters = self.spec.last_query.extra_parameters
```

Comment:
along the same lines here: can one check that query parameters are only allowed to have `shared ` for this package?